internal/connect/command_custom_plugin.go

```diff
@@ -1,26 +1,40 @@
 package connect
 
 import (
+	"sort"
+	"strings"
+
 	"github.com/spf13/cobra"
 
+	connectcustompluginv1 "github.com/confluentinc/ccloud-sdk-go-v2/connect-custom-plugin/v1"
+
 	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
-	v1 "github.com/confluentinc/cli/v3/pkg/config"
+	"github.com/confluentinc/cli/v3/pkg/output"
 )
 
 type customPluginCommand struct {
 	*pcmd.AuthenticatedCLICommand
 }
 
-type customPluginOut struct {
-	Id                  string   `human:"ID" serialized:"id"`
-	Name                string   `human:"Name" serialized:"name"`
-	Description         string   `human:"Description" serialized:"description"`
-	ConnectorClass      string   `human:"Connector Class" serialized:"connector_class"`
-	ConnectorType       string   `human:"Connector Type" serialized:"connector_type"`
-	SensitiveProperties []string `human:"Sensitive Properties" serialized:"sensitive_properties"`
+type customPluginSerializedOut struct {
+	Id                  string   `serialized:"id"`
+	Name                string   `serialized:"name"`
+	Description         string   `serialized:"description"`
+	ConnectorClass      string   `serialized:"connector_class"`
+	ConnectorType       string   `serialized:"connector_type"`
+	SensitiveProperties []string `serialized:"sensitive_properties"`
 }
 
-func newCustomPluginCommand(cfg *v1.Config, prerunner pcmd.PreRunner) *cobra.Command {
+type customPluginHumanOut struct {
+	Id                  string `human:"ID"`
+	Name                string `human:"Name"`
+	Description         string `human:"Description"`
+	ConnectorClass      string `human:"Connector Class"`
+	ConnectorType       string `human:"Connector Type"`
+	SensitiveProperties string `human:"Sensitive Properties"`
```

Comment:
Actually majority of use cases will have this set and we had a big incident related to users not using this so I think it would be nice for them to see that they did not set any sensitive properties that they should have